fuzzing/fuzzer_test.go

```diff
@@ -88,6 +89,7 @@ func TestAssertionMode(t *testing.T) {
 				config.Fuzzing.Testing.AssertionTesting.PanicCodeConfig.FailOnIncorrectStorageAccess = true
 				config.Fuzzing.Testing.AssertionTesting.PanicCodeConfig.FailOnOutOfBoundsArrayAccess = true
 				config.Fuzzing.Testing.AssertionTesting.PanicCodeConfig.FailOnPopEmptyArray = true
+				config.Fuzzing.Testing.AssertionTesting.TestViewMethods = true
```

Comment:
I don't see where `assert_constant_method.sol` is being tested in `fuzzer_test.go`? Maybe you planned on adding it to the `filePaths` variable above?

Reply:
Good catch! https://github.com/crytic/medusa/pull/363/commits/b8fa7f3dbde64b13c919c2230a77674cada33c27